media_kit_video/lib/src/video/video_texture.dart

```diff
@@ -133,19 +138,39 @@ class VideoState extends State<Video> {
     return media_kit_video_controls.toggleFullscreen(context);
   }
 
+  void setSubtitleViewPadding(
+    EdgeInsets padding, {
+    Duration duration = const Duration(milliseconds: 100),
+  }) {
+    return _subtitleViewKey.currentState?.setPadding(
+      padding,
+      duration: duration,
+    );
+  }
+
   @override
   void initState() {
     super.initState();
     if (widget.wakelock) {
-      WakelockPlus.enable().catchError((_) {});
+      if (widget.controller.player.state.playing) {
```

Comment:
I think we must acquire wake-lock if `Player` is already in `playing` state before `Video` mounts.